code/datums/status_effects/debuffs.dm

```diff
@@ -913,6 +913,14 @@
 	examine_text = "<span class='warning'>SUBJECTPRONOUN has a blank, catatonic like stare.</span>"
 	alert_type = /atom/movable/screen/alert/status_effect/ghoul
 
+/datum/status_effect/ghoul/get_examine_text()
+	var/mob/living/carbon/human/H = owner
+	var obscured = H.check_obscured_slots()
+	if(!(obscured & ITEM_SLOT_EYES) && !H.glasses) //The examine text is only displayed if the ghoul's eyes are not obscured
+		return examine_text
+	else
+		return null
```

Comment:
This is redundant

Reply:
Fixed in https://github.com/BeeStation/BeeStation-Hornet/pull/8268/commits/b3487dba30dcd6b74b8dcf553bbcda81bdf19099